### PR TITLE
Reorganize README

### PR DESCRIPTION
**Pull request by tommy-mitchell**
README changes from #3. I kept out the things you said were internal and have left out references to using TypeScript (pending #5 and me opening a PR for [types](https://github.com/tommy-mitchell/supertape/tree/pr/types)). Fixes some typos and improves the structure compared to [before](https://github.com/tommy-mitchell/supertape).

### CODE REVIEW

**Comment by tommy-mitchell** [comment]
I've also added all test options to `supertape.d.ts` to get that out while I'm still working on the other PR.

File: README.md [hunk 1]
```diff
@@ -14,15 +14,18 @@
 
 [**Tape**](https://github.com/substack/tape)-inspired [`TAP`](https://testanything.org/)-compatible simplest high speed test runner with superpowers.
 
+----
+
```
Comment (coderaiser) [inline README.md]: It broken syntax highlighting in VIM

File: README.md [hunk 2]
```diff
@@ -14,15 +14,18 @@
...
 📼 **Supertape** is a fast, minimal test runner with the soul of **tape**. It's designed to be as compatible as possible with **tape** while still having some key improvements, such as:
 
-- the ability to work with [ESM Modules](https://nodejs.org/api/esm.html) (take a look at [mock-import](https://github.com/coderaiser/mock-import) for mocking and 🎩[ESCover](https://github.com/coderaiser/escover) for coverage)
-- a number of [built-in pretty output formatters](#formatters)
-- the ability to [extend](#testextendextensions)
+- out-of-the-box [ES6 module](https://nodejs.org/api/esm.html) support (take a look at [mock-import](https://github.com/coderaiser/mock-import) for mocking and 🎩[ESCover](https://github.com/coderaiser/escover) for coverage)
```
Comment (coderaiser) [inline README.md]:
```suggestion
- out-of-the-box [EcmaScript Modules](https://nodejs.org/api/esm.html) support (take a look at [mock-import](https://github.com/coderaiser/mock-import) for mocking and 🎩[ESCover](https://github.com/coderaiser/escover) for coverage)
```

File: README.md [hunk 3]
```diff
@@ -32,21 +35,64 @@
...
 - `t.throws()`, `t.doesNotThrow()` - use [**tryCatch**](https://github.com/coderaiser/try-catch) or [**tryToCatch**](https://github.com/coderaiser/try-to-catch) with [`t.equal()`](#tequalresult-any-expected-any-message-string) instead
 - [`t.plan()`](https://github.com/substack/tape#tplann)
 
 For a list of all built-in assertions, see [Operators](#operators).
 
-## Install
+## Installation
 
-```
+```plain
```
Comment (coderaiser) [inline README.md]:
```suggestion
```
```

File: README.md [hunk 4]
```diff
@@ -58,152 +104,394 @@ Options
    --no-check-duplicates       do not check messages for duplicates
 ```
 
-## Environment variables
+#### `--version` (alias: `-v`)
```
Comment (coderaiser) [inline README.md:104]: How this part is better then `—help`? Maybe let’s keep `—-help`, so there was one point of truth?

File: README.md [hunk 5]
```diff
@@ -58,152 +104,394 @@ Options
...
+📼 **Supertape** expects that each test contains exactly one assertion. This check can be disabled with:
 
-- passing `--no-check-assertions-count` command line flag;
-- setting `SUPERTAPE_CHECK_ASSERTIONS_COUNT=0` env variable;
+- the [`--no-check-assertions-count`](#--no-check-assertions-count) command line flag
+- the [`SUPERTAPE_CHECK_ASSERTIONS_COUNT`](#supertape_check_assertions_count) environment variable (set to `0`)
+- the [`checkAssertionsCount`](#checkassertionscount-boolean) test option (set to `true`)
 
-#### ❌ Example of incorrect code
+#### ❌ Example of Incorrect Code:
```
Comment (coderaiser) [inline README.md:404]: Could you please lowercase “(in)correct code”. All examples in 🐊[**Putout**](https://github.com/coderaiser/putout) use this format, and would be great to use one format, to simplify migrations.

File: packages/supertape/lib/supertape.d.ts [hunk 6]
```diff
@@ -33,9 +33,15 @@ type Test = Operator & OperatorStub & {
...
+    skip?: boolean;
+    only?: boolean;
+    extensions?: CustomOperator;
+    quiet?: boolean;
+    format?: string;
+    run?: boolean;
+    checkDuplicates?: boolean;
+    checkAssertionsCount?: boolean;
+    checkScopes?: boolean;
 };
```
Comment (coderaiser) [inline packages/supertape/lib/supertape.d.ts:45]: Let’s keep this fields private

**Comment by coderaiser** [comment]
Any progress on this?

File: README.md (hunk 1 above)
Comment (coderaiser) [inline README.md]:
```suggestion
```

**Pull request closed without merging**